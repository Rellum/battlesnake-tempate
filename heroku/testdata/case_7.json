{
  "game": {
    "id": "20256",
    "ruleset": {
      "name": "standard",
      "version": "v.1.2.3"
    },
    "timeout": 500
  },
  "turn": 200,
  "you": {
    "health": 100,
    "id": "you",
    "name": "#22aa34",
    "body": [
      {
        "x": 3,
        "y": 10
      },
      {
        "x": 0,
        "y": 9
      },
      {
        "x": 0,
        "y": 8
      },
      {
        "x": 1,
        "y": 8
      },
      {
        "x": 2,
        "y": 8
      },
      {
        "x": 3,
        "y": 8
      },
      {
        "x": 3,
        "y": 9
      }
    ],
    "head": {
      "x": 3,
      "y": 10
    },
    "length": 7
  },
  "board": {
    "food": [
      {
        "x": 1,
        "y": 10
      },
      {
        "x": 8,
        "y": 7
      },
      {
        "x": 5,
        "y": 5
      },
      {
        "x": 5,
        "y": 2
      },
      {
        "x": 2,
        "y": 2
      },
      {
        "x": 9,
        "y": 1
      }
    ],
    "height": 11,
    "width": 11,
    "snakes": [
      {
        "health": 100,
        "id": "you",
        "name": "#22aa34",
        "body": [
          {
            "x": 3,
            "y": 10
          },
          {
            "x": 0,
            "y": 9
          },
          {
            "x": 0,
            "y": 8
          },
          {
            "x": 1,
            "y": 8
          },
          {
            "x": 2,
            "y": 8
          },
          {
            "x": 3,
            "y": 8
          },
          {
            "x": 3,
            "y": 9
          }
        ],
        "head": {
          "x": 3,
          "y": 10
        },
        "length": 7
      }
    ]
  }
}