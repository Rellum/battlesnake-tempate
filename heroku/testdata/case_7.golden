Food ⚕: [{1 10} {8 7} {5 5} {5 2} {2 2} {9 1}]
#22aa34 ■□□: {you #22aa34 100 [{3 10} {0 9} {0 8} {1 8} {2 8} {3 8} {3 9}] {3 10} 7   }
◦⚕◦■◦◦◦◦◦◦◦
□◦◦□◦◦◦◦◦◦◦
□□□□◦◦◦◦◦◦◦
◦◦◦◦◦◦◦◦⚕◦◦
◦◦◦◦◦◦◦◦◦◦◦
◦◦◦◦◦⚕◦◦◦◦◦
◦◦◦◦◦◦◦◦◦◦◦
◦◦◦◦◦◦◦◦◦◦◦
◦◦⚕◦◦⚕◦◦◦◦◦
◦◦◦◦◦◦◦◦◦⚕◦
◦◦◦◦◦◦◦◦◦◦◦
